posthog/hogql_queries/web_analytics/web_overview.py

```diff
@@ -275,13 +302,12 @@ def to_data(
         if previous is not None:
             previous = previous / 1000
 
-    try:
-        if value is not None and previous:
+    change_from_previous_pct = None
+    if value is not None and previous is not None and previous != 0:
+        try:
             change_from_previous_pct = round(100 * (value - previous) / previous)
-        else:
-            change_from_previous_pct = None
-    except ValueError:
-        change_from_previous_pct = None
+        except (ValueError, ZeroDivisionError):
```

Comment:
Not sure how `ZeroDivisionError` could occur given your check above

Reply:
Yeah, changed this one later